tests/test_text.py

```diff
@@ -168,3 +169,30 @@ def test_text_zfill(data, fletcher_variant):
     # Pandas returns np.nan for NA values in cat, keep this in line
     result_fr[result_fr.isna()] = np.nan
     tm.assert_series_equal(result_fr, result_pd)
+
+
+@settings(deadline=timedelta(milliseconds=1000))
```

Comment:
Deadline should be disabled, otherwise this gets flaky when run in parallel with other builds.
```suggestion
@settings(deadline=None)
```

Reply:
Done